src/components/LeftBar/LeftBar.tsx

```diff
@@ -0,0 +1,29 @@
+import { NavLink } from 'react-router-dom';
+import { List } from '@mui/material';
+
+import { Icon } from 'components/Icon';
+
```

Comment:
Could you please remove this empty line and add one between variables and links?